airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/jinja.py

```diff
@@ -3,32 +3,17 @@
 #
 
 import ast
-import datetime
-import numbers
 
 from airbyte_cdk.sources.declarative.interpolation.interpolation import Interpolation
-from dateutil import parser
+from airbyte_cdk.sources.declarative.interpolation.macros import macros
 from jinja2 import Environment
 from jinja2.exceptions import UndefinedError
 
 
 class JinjaInterpolation(Interpolation):
     def __init__(self):
         self._environment = Environment()
-        # Defines some utility methods that can be called from template strings
-        # eg "{{ today_utc() }}
-        self._environment.globals["now_local"] = datetime.datetime.now
-        self._environment.globals["now_utc"] = lambda: datetime.datetime.now(datetime.timezone.utc)
-        self._environment.globals["today_utc"] = lambda: datetime.datetime.now(datetime.timezone.utc).date()
-        self._environment.globals["timestamp"] = (
-            lambda dt: int(dt)
-            if isinstance(dt, numbers.Number)
-            else int(parser.parse(dt).replace(tzinfo=datetime.timezone.utc).timestamp())
-        )
-        self._environment.globals["max"] = lambda a, b: max(a, b)
-        self._environment.globals["day_delta"] = lambda x: (
-            datetime.datetime.now(datetime.timezone.utc) + datetime.timedelta(days=x)
-        ).strftime("%Y-%m-%dT%H:%M:%S.%f%z")
+        self._environment.globals.update(**{f.__name__: f for f in macros})
```

Comment:
maybe define the macros in `macros.py` as a dictionary and just set this variable to that export? this way it's much clearer that changing a function name is a breaking change. Also add a unit test which asserts that specific function names are exported from that dictionary

Reply:
good idea! tests are in https://github.com/airbytehq/airbyte/pull/14515/files#diff-72d3874dde65f490198cd79e01d527418ade7236353623ad0607e5baf6be84dcR8-R23